### ISSUE TITLE
[JavaScript] get keyword wrongly scoped as storage.type.accessor.js

### ISSUE THREAD

**Issue by rchl** [sublimehq/Packages#379]
```
'use strict';

class Name {
  static get() {
    let foo = 1
  }
```

'get' is treated as storage.type.accessor.js while in this context it should be just a variable.function.js.
Due to this, everything inside the block gets wrong scope.
Probably same problem exists for 'set'.

**Comment by keith-hall** [comment]
https://github.com/sublimehq/Packages/pull/405 includes a fix for this

**Comment by wbond** [comment]
This was fixed by #405 (3a1ad54ff7b6aa22a5d601d9631504a5b90b6f14)
